test/misc.test

```diff
@@ -104,6 +104,14 @@ test_expect_success fiptool "fip" "
 	fiptool info images/test.fip
 "
 
+exec_test_set_prereq mdadm
+test_expect_success mdadm "mdraid" "
+	run_genimage_root mdraid.config test.mdraid &&
+	mdadm --examine images/test.mdraid &&
```

Comment:
Save the output here in a file, and just grep the file below.

Hmm, or better use `test_cmp` to compare the output with a file. With fixed uuids and timestamp, the output should be reproducible.

Reply:
I guess we can do that, but i didn't really wanted to be dependend on exact MDADM version... Hence checking individual fields in the output rather than comparing whole thing, so it keeps working in case they add field or something...

Reply:
yeah. i've tried this and it does not work. mdadm output is not consistent even between both ubuntu versions used in pipeline...